packages/EasyErrorHandler/src/Bridge/Symfony/Commands/AnalyzeErrorCodesCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Commands;
+
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class AnalyzeErrorCodesCommand extends Command
+{
+    /**
+     * @var int
+     */
+    private const DEFAULT_CATEGORY_SIZE = 100;
+
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'error-codes:analyze';
+
+    /**
+     * @var int
+     */
+    private $categorySize;
+
+    /**
+     * @var \EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface
+     */
+    private $errorCodesProvider;
+
+    public function __construct(ErrorCodesProviderInterface $errorCodesProvider, ?int $categorySize = null)
```

Comment:
Please use the property promotion feature here.

Reply:
Only PHP 8+ supports the feature, doesn't it?

Reply:
@voodooism `master` is v4 which is restricted to PHP 8.1+ :wink:

Reply:
In that case, I think we can use strict types too in this PR ?

Reply:
Refactored.